### PR TITLE
Add guidance on using turquoise backgrounds

### PR DESCRIPTION
**Pull request by sanjaypoyzer**
Fixes the problem discussed in issue #200.

### CODE REVIEW

File: views/guide_colour.html [hunk 1]
```diff
@@ -379,6 +379,10 @@ <h4 class="heading-small">GOV.UK blue</h4>
 
   <h3 class="heading-medium" id="examples">Examples</h3>
   <h4 class="heading-small">Transaction end page</h4>
+  
+  <p>
+    Note that the white text on turquoise background only passes accessibility standards if the font size is at least 18px, or 14px bold. 
+  </p>
```
Comment (gemmaleigh) [inline views/guide_colour.html]: 'accessibility standards' is a little vague, could this be revised to "only meets [
[WCAG2 Level AA](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast), with a contrast ratio of 3:1 for large text - when the font size is at least 18px, or 14px bold.

**Comment by sanjaypoyzer** [comment]
@gemmaleigh Not quite sure why the CI failed there...

**Comment by gemmaleigh** [comment]
I'd like to revise this to:

Make sure text on a coloured background is legible.

To meet <a href="https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast">WCAG2 Level AA</a>, you need to make sure the:

- font size is at least 18px, or 14px bold
- contrast ratio is 3:1

--

I've run this past the content designers on the Service Manual team.

@sanjaypoyzer let me know if you're happy to amend and squash those two commits, otherwise I can.

**Comment by edwardhorsford** [comment]
@gemmaleigh Does that mean that essentially services shouldn't use `core-19` in those boxes? (since the font size goes down to 16px on mobile).

**Comment by sanjaypoyzer** [comment]
Yeah sure, that looks clearer. Will just wait for the clarification on the `core-19` point before doing it.

**Comment by gemmaleigh** [comment]
Yes, the example as-is would fail WCAG2 Level AA for smaller viewports, where `.core-19` for body text at 19px decreases to 16px.

If we don't intend to change the colour of the background - then the font size can either be increased, or the 19px white text on a turquoise background could use `.bold-19` which would pass.

**Comment by sanjaypoyzer** [comment]
Should we add that to the guidance as well, or do you think it's clear enough to be inferred?

**Comment by gemmaleigh** [comment]
This was added in #481, closing as this has now been resolved.

**Pull request closed without merging**